<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconTracksEditor</class>
 <widget class="QWidget" name="IconTracksEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelCoverArt">
        <property name="text">
         <string>cover</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFavorite">
        <property name="text">
         <string>favorite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRating">
        <property name="text">
         <string>rating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCount">
        <property name="text">
         <string>count</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSubTitle">
        <property name="text">
         <string>subtitle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
